lib/ansible/modules/network/skydive/skydive_edge.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: skydive_edge
+version_added: "2.8"
+author:
+  - "Sumit Jaiswal (@sjaiswal)"
+short_description: Module to add edges to Skydive topology
+description:
+  - This module handles setting up edges between two nodes based on the
+    relationship type to the Skydive topology.
+requirements:
+  - skydive-client
+extends_documentation_fragment: skydive
+options:
+  parent_node:
+    description:
+      - To defined the first node of the link, it can be either an ID or
```

Comment:
with the current implementation it can't be a gremlin expression, please see

https://github.com/skydive-project/skydive/blob/master/contrib/ansible/library/skydive_edge.py#L168
https://github.com/skydive-project/skydive/blob/master/contrib/ansible/library/skydive_edge.py#L214

Reply:
Done and updated